### PR TITLE
feat(STN-107):  more utilities

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
- new utility classes for:
- blockquote
- divider
- descriptor
- inline group block

## Steps to Test
1. npm run test passes
2. Add new classes to different fields and group blocks:

```
.hb-divider
.hb-views-divider (for views only)
.hb-blockquote
.hb-descriptor
.hb-borderless
.hb-inline
.hb-inline-pipe
```

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss [hunk 1]
```diff
@@ -12,10 +12,67 @@
...
+.hb-inline-pipe {
+  > div {
+    padding-right: 0;
+
+    @include grid-media-min('sm') {
+      display: flex;
+    }
+
+    &::after {
+      @include grid-media-min('sm') {
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss:41]: I like how it drops the pipes and stacks on mobile! 🏆

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss [hunk 2]
```diff
@@ -96,3 +96,12 @@
...
 }
+
+@mixin hb-divider {
+  width: 100%;
+
+  @include hb-colorful {
+    border-bottom: $hb-thin-border;
+    @include hb-global-color('border-color', 'gray');
+  }
+}
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss:107]: Let's update this divider class to apply to views, like we discussed in our call. Also updating the structured card as Alex described. 👍

**Review by bryanbraun — CHANGES_REQUESTED** [review]
These were looking pretty great locally.

Let's revisit the divider one like we had previously discussed, but the rest are looking really good.

(note: there's a failing sass test but I think it'll pass if you rebase this on the sprint branch 👍)

File: docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss (hunk 1 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss:41]: all thanks to @meranicosme

**Review by bryanbraun — APPROVED** [review]

**Pull request merged**